File: walk-time.js
var hrs = Math.floor(Math.random() * 3 + 1);
var mins = Math.floor(Math.random() * 60);

if (mins >= 10) {
  console.log(hrs + ":" + mins + "PM");
} else {
  console.log(hrs + ":0" + mins + "PM");
}

File: walk-time.dart
import 'dart:math';

main() {
  var rng = new Random();
  var hrs = (rng.nextInt(3) + 1).toString();
  var mins = ("0" + rng.nextInt(60).toString());

  print(hrs+":"+mins.substring(mins.length-2)+"PM");
}
